From 45866160b893bacef53e4ea4a2749c3081bd7655 Mon Sep 17 00:00:00 2001
From: broodplank <guichardflorent@gmail.com>
Date: Thu, 19 Dec 2013 00:15:21 +0100
Subject: [PATCH 08/18] core: added national roaming info for Tesco Mobile in
 Ireland.

Users of Tesco Mobile in Ireland(MCC 272 MNC 11)
  can roam on O2 Ireland (MNC 02)

Change-Id: Ie0b32d51511c2bbc726c43cdec78332397d1ecb9
---
 core/res/res/values-mcc272-mnc11/config.xml | 25 +++++++++++++++++++++++++
 1 file changed, 25 insertions(+)
 create mode 100644 core/res/res/values-mcc272-mnc11/config.xml

diff --git a/core/res/res/values-mcc272-mnc11/config.xml b/core/res/res/values-mcc272-mnc11/config.xml
new file mode 100644
index 0000000..f7850c9
--- /dev/null
+++ b/core/res/res/values-mcc272-mnc11/config.xml
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+/*
+** Copyright 2013, The Android Open Source Project
+**
+** Licensed under the Apache License, Version 2.0 (the "License");
+** you may not use this file except in compliance with the License.
+** You may obtain a copy of the License at
+**
+**     http://www.apache.org/licenses/LICENSE-2.0
+**
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+-->
+
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+    <!-- Don't use roaming icon for considered operators -->
+    <string-array translatable="false" name="config_operatorConsideredNonRoaming">
+        <item>27202</item>
+    </string-array>
+</resources>
-- 
2.7.4

